pkg/reportingplugins/mercury/v0/types.go

```diff
@@ -0,0 +1,41 @@
+package mercury_v0
+
+import (
+	"github.com/smartcontractkit/chainlink-relay/pkg/reportingplugins/mercury"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+type IParsedAttributedObservation interface {
```

Comment:
Just personal preference here, but how about calling this simply `ParsedAttributedObservation` and making the concrete struct private, i.e. `parsedAttributedObservation`?

Reply:
Sure we can do it this way. I wanted to differentiate between struct and interface when splitting the functionality but there is no need for that anymore.